app/views/projects/index.html.erb

```diff
@@ -9,9 +9,10 @@
 <div class="center">
 <h2>Projects</h2>
   <%= search_form_for @search do |f| %>
+      <%= f.label :badge_status %>
+      <%= f.collection_select(:badge_status_matches, Project::BADGE_STATUS_CHOICE, :id, :name) %>
```

Comment:
`collection_select` is for ActiveRecord models. Instead, you can use:
```ruby
select_tag(:badge_status_matches, options_for_select(Project::BADGE_STATUSES))
```
If you want to play with this from `rails console`, you can do:
```ruby
helper.select_tag(:badge_status_matches, helper.options_for_select([nil, "in_progress", "passing", "failing"]))
```